packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -112,6 +112,12 @@ class ImagePicker {
   /// image types such as JPEG and on Android PNG and WebP, too. If compression is not
   /// supported for the image that is picked, a warning message will be logged.
   ///
+  /// The `limit` parameter modifies the maximum number of images that can be selected.
+  /// On Android, you need to set ImagePickerAndroid's useAndroidPhotoPicker to
```

Comment:
There should be `[]` around `ImagePickerAndroid`, and ` `` ` around `useAndroidPhotoPicker`

This also seems a little verbose, maybe shorten it to 

```On Android, [ImagePickerAndroid.useAndroidPhotoPicker] must be set to `true` to use the `limit` functionality```

I'm not sure the `see: [link]` part is necessary either.

Same for the other instance of this comment.

Reply:
Thanks for the review, I've readjusted it.